<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportExportDialog</class>
 <widget class="QWidget" name="ImportExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import/Export Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="m_modeGroupBox">
     <property name="title">
      <string>Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="m_exportButton">
        <property name="text">
         <string>Export Settings</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_importButton">
        <property name="text">
         <string>Import Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_importAndApplyButton">
        <property name="text">
         <string>Import Settings and Apply to all Windows</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_settingsGroup">
     <property name="title">
      <string>Settings to Export</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="m_bookViewSettingsButton">
        <property name="text">
         <string>Book View</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_dashboardsButton">
        <property name="text">
         <string>Dashboards</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_orderImbalanceIndicatorButton">
        <property name="text">
         <string>Order Imbalance Indicator</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_interactionsButton">
        <property name="text">
         <string>Interactions</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_keyBindingsButton">
        <property name="text">
         <string>Key Bindings</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_portfolioViewerSettingsButton">
        <property name="text">
         <string>Portfolio Viewer</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_timeAndSalesSettingsButton">
        <property name="text">
         <string>Time and Sales</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_windowLayoutButton">
        <property name="text">
         <string>Window Layout</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="m_pathLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_pathLabel">
       <property name="text">
        <string>Settings Path:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Spire::UI::FilePathInputWidget" name="m_pathInput" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Spire::UI::FilePathInputWidget</class>
   <extends>QWidget</extends>
   <header>Spire/UI/FilePathInputWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>CommitSettings()</slot>
 </slots>
</ui>
